service/src/main/java/bio/terra/pipelines/app/controller/PipelineRunsApiController.java

```diff
@@ -217,6 +217,8 @@ public ResponseEntity<ApiGetPipelineRunsResponse> getAllPipelineRuns(
         pipelinesService.getPipelines().stream()
             .collect(Collectors.toMap(Pipeline::getId, pipeline -> pipeline.getName().getValue()));
 
+    int totalResults = Math.toIntExact(pipelineRunsService.getPipelineRunCount(userId));
```

Comment:
do you get some kind of weird typing error if you just pass this along as a long?